test/unit/support.js

```diff
@@ -58,53 +58,17 @@ testIframe(
 	var expected,
 		userAgent = window.navigator.userAgent,
 		expectedMap = {
-			edge: {
-				boxSizingReliable: true,
-				clearCloneStyle: true,
-				noCloneChecked: true,
-				optSelected: true,
-				radioValue: true
-			},
-			ie_11: {
-				boxSizingReliable: false,
-				clearCloneStyle: false,
-				noCloneChecked: false,
-				optSelected: false,
-				radioValue: false
-			},
-			chrome: {
-				boxSizingReliable: true,
-				clearCloneStyle: true,
-				noCloneChecked: true,
-				optSelected: true,
-				radioValue: true
-			},
-			safari: {
-				boxSizingReliable: true,
-				clearCloneStyle: true,
-				noCloneChecked: true,
-				optSelected: true,
-				radioValue: true
-			},
-			firefox: {
-				boxSizingReliable: true,
-				clearCloneStyle: true,
-				noCloneChecked: true,
-				optSelected: true,
-				radioValue: true
-			},
-			ios: {
-				boxSizingReliable: true,
-				clearCloneStyle: true,
-				noCloneChecked: true,
-				optSelected: true,
-				radioValue: true
-			}
+			edge: {},
+			ie_11: {},
+			chrome: {},
+			safari: {},
+			firefox: {},
+			ios: {}
 		};
 
 	if ( /edge\//i.test( userAgent ) ) {
 		expected = expectedMap.edge;
-	} else if ( document.documentMode ) {
+	} else if ( /trident\//i.test( userAgent ) ) {
```

Comment:
I had to change this check back to be UA-based as Chrome with faked `documentMode` would still pass the support tests that IE fails.

There's no difference at the moment but if we had to add some the problem would surface.